steering_committee.md

```diff
@@ -5,123 +5,57 @@ The FOCUS SC will be responsible for all oversight of the open source projects.
 
 | Name                                               | Affiliation           | Term begins | Term ends    |
 | -------------------------------------------------- | --------------------- | ----------- | ------------ |
-| Mike Fuller | Linux Foundation | 3/29/2023 | 3/29/2025 |
-| Michael Flanakin | Microsoft | 3/29/2023 | 3/29/2025 |
-| Eleni Rundle | Google | 3/29/2023 | 3/29/2025 |
+| Mike Fuller                                        | Linux Foundation      | 3/29/2023   | 3/29/2025    |
+| Michael Flanakin                                   | Microsoft             | 3/29/2023   | 3/29/2025    |
+| Eleni Rundle                                       | Google                | 3/29/2023   | 3/29/2025    |
 
 #### Emeriti
 
 We thank all our prior SC members for their contribution:
 
 | Name                                               | Affiliation           | Term began  | Term ended   |
 | -------------------------------------------------- | --------------------- | ----------- | ------------ |
-| - | - | - | - |
+| -                                                  | -                     | -           | -            |
 
-### Becoming a SC Member
 
-The [Charter](FOCUS_-_Membership_Agreement_Package_for_use.pdf) describes the composition of the SC. Membership levels of Member organizations are managed by the FinOps Foundation and Steering Committee memberships are voted on and approved by the FinOps Foundation Governing Board.
+One of the more important duties of the Steering Committee (SC) is the approval of the Specifications and other works produced as a consensus product of the FOCUS Working Groups.
 
-## Policies and procedures
+  The Steering Committee is initially composed of a representative of the founding members of the Organization and it has a single primary member representing each company. Additional SC members will be appointed by the FinOps Foundation Governing Board as specified in the FOCUS Project Governance document.
 
-The FOCUS SC is governed by the [Charter](FOCUS_-_Membership_Agreement_Package_for_use.pdf). The Charter provides a foundational structure for the SC on topics such as its scope, how to make decisions, and how to make changes to itself.  At the same time, it grants the SC a high degree of freedom when determining how to implement the policies of FOCUS.
+  Each Steering Committee meeting is called on a regular interval, although this interval can be ad-hoc as long as the proper notice is given.
 
-The following policies and procedures have been adopted by the SC.
+  Proper notice of the Steering Committee (SC) meeting is given to its representatives including an agenda with the topics to be voted by the SC having been prepared with a proper notice period - 7 days.
 
-### Making changes to this repository
+  A meeting of the Steering Committee should have a quorum of more than one half of currently serving Steering Committee participants for any call to vote to proceed.
```

Comment:
This is worded a bit funny. "A quorum of 51% of the Steering Committee is required for a vote to proceed."